src/sentry/static/sentry/app/views/settings/projectAlerts/index.tsx

```diff
@@ -10,23 +10,28 @@ type Props = {
   children: React.ReactNode;
 } & RouteComponentProps<{organizationId: string; projectId: string}, {}>;
 
-const ProjectAlerts = ({children, organization}: Props) => (
-  <Access organization={organization} access={['project:write']}>
-    {({hasAccess}) => (
-      <Feature organization={organization} features={['incidents']}>
-        {({hasFeature}) => (
-          <React.Fragment>
-            {React.isValidElement(children) &&
-              React.cloneElement(children, {
-                organization,
-                canEditRule: hasAccess,
-                hasMetricAlerts: hasFeature,
-              })}
-          </React.Fragment>
-        )}
-      </Feature>
-    )}
-  </Access>
-);
+const ProjectAlerts = ({children, organization}: Props) => {
+  const hasCreateFromDiscover = organization.features.includes('create-from-discover');
```

Comment:
Wasn't sure if this was the best way to handle this. I don't want to hide anything if they don't have the feature.

Reply:
You could do something like https://github.com/getsentry/sentry/pull/19381/files#diff-d67a1699447f1a63c7f2425550031985R19

Reply:
Changed to use projects